src/main/kotlin/nebula/plugin/bintray/BintrayService.kt

```diff
@@ -0,0 +1,40 @@
+package nebula.plugin.bintray
+
+import retrofit2.Call
+import retrofit2.http.*
+
+interface BintrayService {
+    @Headers("Content-Type: application/json")
+    @POST("/content/{subject}/{repo}/{pkg}/{version}/publish")
```

Comment:
This would actually be for the [repository url definition](https://github.com/nebula-plugins/nebula-bintray-plugin/pull/29/files#diff-f0b0aab6558824702b67652c840f2af4R61)

Reply:
Will do it in https://github.com/nebula-plugins/nebula-bintray-plugin/pull/29/files#diff-f0b0aab6558824702b67652c840f2af4R61